### DISCUSSION TITLE
Increase map resolution

### DISCUSSION THREAD

**Discussion by arne182** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#637] [Q&A]
![Screenshot_20230727_111043_Home Assistant](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/844102/a6ba08a3-65aa-41e6-acca-45364d1b10e8)
![Screenshot_20230727_111152_Roborock](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/844102/069038a7-8e77-4971-a588-e2a16eef3ff8)
Is there a possibility to increase the resolution of the map? In comparison to the original it is looking quite pixelated.

**Comment by PiotrMachowski** [comment]
This card doesn't create the map, it just displays it, so it can't do anything to it.

If you use Map Extractor you can change "scale" parameter to increase the map resolution (2 should be enough).
